account-kit/signer/src/session/manager.ts

```diff
@@ -230,28 +244,45 @@ export class SessionManager {
       }
     );
 
-    this.client.on("disconnected", () => this.clearSession());
-
-    this.client.on("connectedEmail", (user, bundle) =>
-      this.setSessionWithUserAndBundle({ type: "email", user, bundle })
-    );
+    // Helper type to ensure that a listener is either defined or explicitly
+    // omitted for every event type.
+    type Listeners = {
```

Comment:
@moldy530 I kept forgetting to add listeners for new events I added so I changed this so the type system would help. Style question: do you prefer using `undefined` for events where we don't want a listener, or the noop function `() => {}`?

Reply:
oh good question... I think undefined is simpler devex, but only slightly. so I don't have a super strong feeling here